Sources/IBMSwiftSDKCore/Authentication/IAMAuthenticator.swift

```diff
@@ -114,7 +126,7 @@ class IAMTokenSource: TokenSource {
         return RestError.http(statusCode: response.statusCode, message: errorMessage, metadata: metadata)
     }
 
-    private func requestToken(completionHandler: @escaping (IAMToken?, RestError?) -> Void) {
+    private func requestToken(completionHandler: @escaping (String?, RestError?) -> Void) {
```

Comment:
I changed the signature on `requestToken` so that the completion handler for `getToken` could be passed directly into this method, as is done in the `CloudPakForDataAuthenticator`.  To accommodate this, we compute and store the refreshDate below, rather than in `getToken`.

This change makes the two authenticators more comparable in logic flow, which should aid in future development.